src/v2/Apps/Article/Components/ArticleHTML.tsx

```diff
@@ -1,13 +1,43 @@
-import { Box, BoxProps, THEME_V3 as THEME } from "@artsy/palette"
+import { Box, BoxProps, THEME_V3 as THEME, useDidMount } from "@artsy/palette"
 import { themeGet } from "@styled-system/theme-get"
 import { FC } from "react"
 import styled from "styled-components"
+import reactHtmlParser from "@artsy/react-html-parser"
```

Comment:
Note to self to add this to the proper deps. I think it's currently a transient dep via reaction.